js/projects.js

```diff
@@ -0,0 +1,168 @@
+              <p>${data.tech[0]}</p>
+            </li>
+            <li class="program-language">
+              <p>${data.tech[1]}</p>
+            </li>
+
+            <li class="program-language-javascript">
+              <p>${data.tech[2]}</p>
+            </li>
+          </ul> 
+
+      <ul class="project-program-laguages-DK">
+            <li class="program-language">
+              <p>${data.tech[0]}</p>
+            </li>
+            <li class="program-language">
+              <p>${data.tech[1]}</p>
+            </li>
+
+            <li class="program-language-javascript">
+              <p>${data.tech[0]}</p>
+            </li>
+          </ul>
+      
+          <button type="submit" class="call-to-action" id="${data.id}">See Project</button>
+
+        </div>
+
+      </section>
+      <div class="workDivider"></div>
+
+    `,
+  )
+  .join('');
+
+const displayPopup = (data) => {
+  const popupSection = document.querySelector('#popups');
+  popupSection.innerHTML = `
+     <div class="modal-container">
+            <div class="modal-head">
+              <h1>${data.title}</h1>
+              <button class="close">&times; </button>
+            </div>
+  
+            <ul class="modal-dec">
+              <li>${data.canopy[0]}</li>
+              <li>
+                <img class="modal-dot" src="./assets/icons/Counter.svg" alt="dot">
+              </li>
+              <li>${data.canopy[1]}</li>
+              <li>
+                <img class="modal-dot" src="./assets/icons/Counter.svg" alt="dot">
+              </li>
+              <li>${data.canopy[2]}</li>
+            </ul>
+            <div class="modal-img-container">
+              <img class="modal-img modal-img-MO" src="${data.imageMob}" alt="project images">
+              <img class="modal-img modal-img-DK" src="${data.imageDesk}" alt="project images">
+            </div>
+            <div class="modal-footer">
+  
+  
+              <div class="modal-text footer-1">${data.para}</div>
+              <div class="footer-2">
+                <ul class="modal-technology">
+                  <li class="tech">
+                    <p>${data.tech[0]}</p>
+                  </li>
+                  <li class="tech">
+                    <p>${data.tech[1]}</p>
+                  </li>
+  
+                  <li class="tech javascript">
+                    <p>${data.tech[2]}</p>
+                  </li>
+                </ul>
+  
+                <div class="modal-button">
+                  <a href="${data.live}" class="modal-link">
+                    <p>See Live</p>
+                    <img src="./assets/icons/Icon.svg" alt="icons">
+                  </a>
+  
+                  <a href="${data.source}" class="modal-link">
+                    <p>See Source</p>
+                    <img src="./assets/icons/Vector.svg" alt="icons">
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
```

Comment:
You have done a great job implementing the popup functionalities👍 but popup window still looks a bit different from the design😉 currently, you haven't implemented the background of the popup window as given by the design. this is making the other contents visible in the background and this can cause distractions, leading to a less better user experience😉😉 also, take note of the height and width of the image and the popup window, the spacings of the contents of the modal , also take note of the positions and spacing of the buttons and skills container🤗🤗  I suggest you use the exact content given in the design😉 

your project
![image](https://user-images.githubusercontent.com/64643995/216380413-8983e83c-b14d-462f-b924-03f164356eb0.png)

design
![image](https://user-images.githubusercontent.com/64643995/216382571-ac699ec0-0393-40d7-af94-e80ce8638aad.png)


 also, I noticed the mobile version of your project is overflowing, this can lead to a frustrating user experience if attention is not given to it😉😉 kindly put these things into consideration and make the necessary changes😉😉

Reply:
thank you for your review. all suggestions has been address